bangazonapi/views/profile.py

```diff
@@ -207,11 +207,11 @@ def cart(self, request):
             """
 
             try:
-                open_order = Order.objects.get(customer=current_user)
+                open_order = Order.objects.get(customer=current_user, payment_type=None)
                 print(open_order)
             except Order.DoesNotExist as ex:
                 open_order = Order()
-                open_order.created_date = datetime.datetime.now()
+                open_order.created_date = date.datetime.now()
```

Comment:
I'm getting a `NameError: name 'date' is not defined` when I test the pr here

Reply:
'date.datetime.now()' works locally but 'datetime.datetime.now()'returns: 
`AssertionError: Expected a 'date', but got a 'datetime'. Refusing to coerce, as this may mean losing timezone information. Use a custom read-only field and deal with timezone issues explicitly.`

Reply:
could be an import or package error?

Reply:
or is this an acceptable/intended error?

Reply:
`date` isn't getting imported in this file so that is not working locally
I see where you were trying to go with the second error. You've just got the order backwards. Should be `datetime.date.now()`

Reply:
New commit added to fix datetime error